core/model_mixins.py

```diff
@@ -20,3 +20,20 @@ def get_contacts_structures_not_in_fin_suivi(self):
 
     def get_message_form(self):
         raise NotImplementedError
+
+    def add_fin_suivi(self, user):
+        with transaction.atomic():
+            fin_suivi_contact = FinSuiviContact(
+                content_object=self,
+                contact=Contact.objects.get(structure=user.agent.structure),
+            )
+            fin_suivi_contact.full_clean()
+            fin_suivi_contact.save()
```

Comment:
```suggestion
            fin_suivi_contact.save(update_fields=("content_object", "contact"))
```

Il semble que depuis Django 4.2, [ce soit une bonne idée de systématiquement utiliser `update_fields`](https://docs.djangoproject.com/en/5.2/releases/4.2/#setting-update-fields-in-model-save-may-now-be-required). Perso j'avoue préférer l'utiliser autant que possible pour toujour avoir la meilleure perf possible.

Reply:
(Pour être honnéte j'ai juste déplacé le code existant)

:thinking: Je ne suis pas sur de comprendre à quoi cela servirait ici puisque je créé l'entièreté de l'objet (tous les champs). A la limite on pourrait même passer par objects.create non ?

Reply:
Ah oui, pardon, c'est moi.

Reply:
J'ai profité de l'autre modif pour passer sur create() comme ça c'est peut être plus explicite au passage.